ratis-server/src/test/java/org/apache/ratis/RaftBasicTests.java

```diff
@@ -337,6 +337,22 @@ public void testWithLoad() throws Exception {
     runWithNewCluster(NUM_SERVERS, cluster -> testWithLoad(10, 300, false, cluster, LOG));
   }
 
+  @Test
+  public void testPauseFollowers() throws Exception {
+    runWithNewCluster(NUM_SERVERS, this::runTestPauseFollowers);
+  }
+
+  void runTestPauseFollowers(CLUSTER cluster) throws Exception {
```

Comment:
The test is too simple for the pause() feature.  The test should keep sending requests.  Then call pause() and make sure the request after paused will not be executed.

Reply:
Sounds good. Have updated this test to also tes resume. Introduced a thread to send messages by a client meanwhile pause/resume a follower to verify that pause/resume works well.